avaframe/com2AB/com2ABCfg.ini

```diff
@@ -0,0 +1,62 @@
+### Config File - This file contains the main settings for the com2AB run
+## Set your parameters
+# This file is part of Avaframe.
+# This file will be overridden by local_com2ABCfg.ini if it exists
+# So copy this file to local_com2ABCfg.ini, adjust your variables there
+
+
+# Required settings------------------------------
+[INPATH]
+ProfileLayer = avaframe/data/avaSlide/LINES/slideProfiles
+DGMSource = avaframe/data/avaSlide/slideTopo.asc
+SplitPointSource = avaframe/data/avaSlide/POINTS/slidePoints
```

Comment:
I'd like to be able to just give an (general) avalanche directory (according to our input specifications) and alphaBeta then figuress out these files by itself, i.e.:

- Whatever is in LINES are the profiles AB uses
- whatever is in POINTS are the splitpoints
- the one ASCII that is in the avalanche root directory is the DGM

and so on. 

If we feel this is too uncertain, i.e. maybe we want points for something else, we could define that the user needs to name it with _AB_ somewhere in the filename, so we check the directory for files with this filename...

This means these variable don't nee to be set here, same goes for the saveOutPath, as this is just AvaLancheRoot/Output or WorkDir/Output

I suggest to also don't provide outputName, but let it be something like
AvalanchenameNameofprofile  
where nameofprofile comes from the attribute in the slideProfiles file or, if not provided there, we use a default ProfileX naming...